### PR TITLE
bc, figlet: add Spanish translation

### PR DESCRIPTION
**Pull request by victorhck**
- [X] The page (if new), does not already exist in the repo.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.es/common/bc.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# bc
+
+> Un lenguaje de calculadora de precisión arbitraria.
+> Más información: <https://manned.org/bc>.
+
+- Iniciar `bc` en el modo interactivo utilizando la biblioteca matemática estándar:
```
Comment (navarroaxel) [inline pages.es/common/bc.md:6]: No es infinitivo:
```suggestion
- Inicia `bc` en el modo interactivo utilizando la biblioteca matemática estándar:
```

File: pages.es/common/bc.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+`bc -l`
+
+- Calcular el resultado de una expresión:
```
Comment (navarroaxel) [inline pages.es/common/bc.md:10]:
```suggestion
- Calcula el resultado de una expresión:
```

File: pages.es/common/bc.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+`bc <<< "(1 + 2) * 2 ^ 2"`
+
+- Calcular el resultado de una expresión y forzar a que el resultado tenga 10 cifras decimales:
```
Comment (navarroaxel) [inline pages.es/common/bc.md:14]:
```suggestion
- Calcula el resultado de una expresión y fuerza a que el resultado tenga 10 cifras decimales:
```

File: pages.es/common/bc.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`bc <<< "scale=10; 5 / 3"`
+
+- Calcular el resultado de una expresión con el seno y coseno utilizando `mathlib`:
```
Comment (navarroaxel) [inline pages.es/common/bc.md:18]:
```suggestion
- Calcula el resultado de una expresión con el seno y coseno utilizando `mathlib`:
```

File: pages.es/common/figlet.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+# figlet
+
+> Genera encabezados usando caracteres ASCII desde la entrada del usuario.
+> Más información: <http://www.figlet.org/figlet-man.html>.
+
+- Generar el encabezado directamente introduciendo el texto:
```
Comment (navarroaxel) [inline pages.es/common/figlet.md:6]:
```suggestion
- Genera el encabezado directamente introduciendo el texto:
```

File: pages.es/common/figlet.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+`figlet {{entrada_de_texto}}`
+
+- Usar un archivo de fuente personalizada:
```
Comment (navarroaxel) [inline pages.es/common/figlet.md:10]:
```suggestion
- Usa un archivo de fuente personalizada:
```

File: pages.es/common/figlet.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# figlet
+
+> Genera encabezados usando caracteres ASCII desde la entrada del usuario.
+> Más información: <http://www.figlet.org/figlet-man.html>.
+
+- Generar el encabezado directamente introduciendo el texto:
+
+`figlet {{entrada_de_texto}}`
```
Comment (navarroaxel) [inline pages.es/common/figlet.md:8]:
```suggestion
`figlet {{texto_de_entrada}}`
```

File: pages.es/common/figlet.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+- Usar un archivo de fuente personalizada:
+
+`figlet {{entrada_de_texto}} -f {{nombre_de_archivo_de_fuente}}`
```
Comment (navarroaxel) [inline pages.es/common/figlet.md:12]: We start using `path/to/font_file` in newer pages, so:
```suggestion
`figlet {{texto_de_entrada}} -f {{ruta/al/archivo_de_fuente}}`
```

File: pages.es/common/figlet.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+- Redirigir la salida de un comando hacia figlet:
```
Comment (navarroaxel) [inline pages.es/common/figlet.md:14]:
```suggestion
- Redirige la salida de un comando hacia figlet:
```

File: pages.es/common/figlet.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
...
+
+`{{comando}} | figlet`
+
+- Mostrar las fuentes disponibles para figlet:
```
Comment (navarroaxel) [inline pages.es/common/figlet.md:18]:
```suggestion
- Muestra las fuentes disponibles para figlet:
```

File: pages.es/common/figlet.md [hunk 11]
```diff
@@ -0,0 +1,20 @@
...
+
+`showfigfonts {{texto_opciona_a_mostrar}}`
```
Comment (navarroaxel) [inline pages.es/common/figlet.md:20]: Not sure why `showfigfonts` is in this page :thinking: 
```suggestion
`showfigfonts {{texto_opcional_a_mostrar}}`
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]
Hi @victorhck, welcome and thanks for your submission. I left some comments.

**Comment by victorhck** [comment]
> Hi @victorhck, welcome and thanks for your submission. I left some comments.
Thanks...
So, sorry for the noise!! Next time hope to do better...

**Review by CleanMachine1 — APPROVED** [review]
Can't speak Spanish however looks alright

File: pages.es/common/figlet.md [hunk 12]
```diff
@@ -0,0 +1,20 @@
...
+
+`figlet {{texto_de_entrada}} -f {{ruta/al/archivo_de_fuente}}`
+
+- Redirige la salida de un comando hacia figlet:
+
+`{{comando}} | figlet`
+
+- Muestra las fuentes disponibles para figlet:
+
+`showfigfonts {{texto_opcional_a_mostrar}}`
```
Comment (CleanMachine1) [inline pages.es/common/figlet.md:20]: I know the en has it however should it be here? (and on the other one)

Comment (victorhck) [reply pages.es/common/figlet.md:20]: Yes, appears in English file, that's why included here.
I think it's related in some way with figlet. And thanks to be included here it's how I found that this command exits! :)
I'm a newcomer here, but I find it useful...

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**